web/packages/teleterm/src/ui/ConnectMyComputer/DocumentConnectMyComputerSetup/DocumentConnectMyComputerSetup.tsx

```diff
@@ -55,19 +55,29 @@ export function DocumentConnectMyComputerSetup(
 }
 
 function Information(props: { onSetUpAgentClick(): void }) {
+  const { rootClusterUri } = useWorkspaceContext();
+  const { clustersService, mainProcessClient } = useAppContext();
+  const cluster = clustersService.findCluster(rootClusterUri);
+  const { username: systemUsername, hostname } =
+    mainProcessClient.getRuntimeSettings();
+
   return (
     <>
-      {/* TODO(gzdunek): change the temporary copy */}
       <Text>
-        The setup process will download the teleport agent and configure it to
-        make your computer available in the <strong>acme.teleport.sh</strong>{' '}
-        cluster.
+        The setup process will download and launch the Teleport agent, making
+        your computer available in the <strong>{cluster.name}</strong> cluster
+        as <strong>{hostname}</strong>.
+        <br />
+        <br />
+        Cluster users with the role{' '}
+        <strong>connect-my-computer-{cluster.loggedInUser.name}</strong> will be
+        able to access your computer as <strong>{systemUsername}</strong>.
```

Comment:
I wanted to suggest that we could move the role and username to the CMC context so I could reuse them in the status page, but I realized it has not been merged yet 😏

Reply:
Initially, I thought of avoiding hardcoding the name of the role altogether. But before the setup starts, there's no way through which we could pass it from tshd to Connect.

We'd need to have a separete RPC for this which I thought would introduce more problems than it would solve.